plugins/package-managers/bazel/src/main/kotlin/BazelModel.kt

```diff
@@ -31,8 +31,18 @@ private val json = Json { ignoreUnknownKeys = true }
  */
 @Serializable
 internal data class Lockfile(
-    val flags: Flags? = null
+    // The flags containing the registry URLs (Bazel < 7.2.0).
```

Comment:
Why have you chosen one single class vs. isolated implementation per lockfile variants?
(There seems to be no overlap at all)

Reply:
This is for the ease of implementation.

If you have different model classes for different versions of the lockfile, how would you read it? 
With some JSON mapper tricks?

Reply:
> If you have different model classes for different versions of the lockfile, how would you read it?

Theoretically, you could do it [via `polymorphicDefaultDeserializer` like we do in `ScanCodeResultParser`](https://github.com/oss-review-toolkit/ort/blob/985b25bb74cd015aa792e10362fb03d7c7e34c5d/plugins/scanners/scancode/src/main/kotlin/ScanCodeResultParser.kt#L45-L64). But I guess that could be overkill for the single property here.